docs/about/contributing.md

```diff
@@ -41,148 +41,158 @@ We use [semantic-release](https://www.npmjs.com/package/semantic-release), which
 
 ### Models
 
-The object models provide the definition of the data that is stored in data stores. This is done in two parts:
+The object models provide the definition of the data that is saved in datastores (analogous to databases). This is done in two parts:
 
-* **[data-schema][dataschema-repo]**: Schema definition with *[Joi](https://www.npmjs.com/package/joi)* [![Version][dataschema-npm-image]][dataschema-npm-url]
-* **[models][models-repo]**: Specific business logic around the data schema [![Version][models-npm-image]][models-npm-url]
+* **[data-schema][dataschema-repo]**: Schema definition with *[Joi](https://www.npmjs.com/package/joi)*
+* **[models][models-repo]**: Specific business logic around the data schema
 
 ### Datastores
 
 A datastore implementation is used as the interface between the API and a data storage mechanism. There are several implementations written in node around a common interface.
 
-* **[datastore-base][datastore-base-repo]**: Common interface [![Version][datastore-base-npm-image]][datastore-base-npm-url]
-* **[datastore-sequelize][datastore-sequelize-repo]**: Mysql, postgres, sqlite3 and mssql implementation [![Version][datastore-sequelize-npm-image]][datastore-sequelize-npm-url]
+* **[datastore-base][datastore-base-repo]**: Base class defining the interface for datastore implementations
+* **[datastore-sequelize][datastore-sequelize-repo]**: MySQL, PostgreSQL, SQLite3, and MS SQL implementations
+* **[datastore-dynamodb][datastore-dynamodb-repo]**: DynamoDB implementation, with [dynamic-dynamodb][dynamic-dynamodb-repo] being used to create the datastore tables for it
+
+### Artifacts
 
-### Scms
+The [Artifact Store][store-repo] (not to be confused with the datastores mentioned above) is used for saving log outputs, shared steps, templates, test coverage, and any artifacts that are generated during a build. The [log service][log-service-repo] is a Go tool for reading logs from the Launcher and uploading them to the store. The [artifact-bookend][artifact-bookend-repo] is used for uploading artifacts to the store.
 
-An scm implementation is used as the interface between the API and an scm. There are several implementations written in node around a common interface.
+### Source Code Management
 
-* **[scm-base][scm-base-repo]**: Common interface [![Version][scm-base-npm-image]][scm-base-npm-url]
-* **[scm-bitbucket][scm-bitbucket-repo]**: Bitbucket implementation [![Version][scm-bitbucket-npm-image]][scm-bitbucket-npm-url]
-* **[scm-github][scm-github-repo]**: Github implementation [![Version][scm-github-npm-image]][scm-github-npm-url]
+An SCM implementation is used as the interface between the API and an SCM. There are several implementations written in node around a common interface.
 
-### [Config Parser][config-parser-repo] [![Version][config-parser-npm-image]][config-parser-npm-url]
+* **[scm-base][scm-base-repo]**: Common interface
```

Comment:
Ahh ok. I think we should leave it out for now then, since there's nothing there for people to contribute on.

And got it, thanks for the explanation. So should it still be listed under this directory? @petey ?